code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -454,6 +454,19 @@
 	tail = "armalis_tail"
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
+	atmos_requirements = list(
```

Comment:
This is just ensuring the Armalis aren't affected by the increased deadliness of Oxygen to their lesser counterparts the Primalis.